### ISSUE TITLE
Chronological collections

### ISSUE THREAD

**Issue by Floppy** [benbalter/jekyll-style-guide#5]
There is a bit of an inconsistency in the way you describe collections...

```
Use collections for any logically grouped content, chronological or not
```

But, in the ELI5 article on collections, you suggest not to use them for content grouped by date (in the flowchart, particularly). Have you changed your mind on using collections for chronological data?

Not asking because I think it's a bug, but because I honestly want to know what you prefer now. I'm using posts and categories for non-post content at the moment, but I'm not happy with it... see https://github.com/SomethingNewUK/somethingnewuk.github.io/tree/master/_posts/elections

**Comment by benbalter** [comment]
> But, in the ELI5 article on collections, you suggest not to use them for content grouped by date (in the flowchart, particularly). Have you changed your mind on using collections for chronological data?

Nice catch. The reason for the logic was that in Jekyll < 3, collections didn't have a date. Starting with Jekyll 3, posts are just an automatically defined collection, meaning posts technically are a collection so the distinction's a bit fuzzy. 

Maybe it'd make sense to use posts if the site's primary chronological content can be called a post, and arbitrary collections for any other non-post grouped content, chronological or not? Another approach could be always use collections for any logically grouped content. If the thing you're grouping is called a post, you don't have to initialize the collection in your config.

**Comment by Floppy** [comment]
Aha, I wondered if it was a Jekyll 3 change, I thought I'd read something on that. The style guide as it stands is how I'd prefer it to work, certainly, and what you suggest makes sense. I'll have a poke around in my site, see how it works in J3, then come back and suggest any text improvements once I know what I'm talking about :)

**Comment by Floppy** [comment]
Having updated my site now, how about this:

If you only have posts, then use the defaults and the global `permalink:` option to specify the path.

```
permalink: /news/:year/:month/:day/:title.html
```

If you have other chronological content, then use another collection as well:

```
collections:
  elections:
    output: true
    permalink: /elections/:year/:month/:day/:title.html
```

(edited from original comment, as I was thinking you could override the global permalink for posts to make the config consistent, but you can't).

**Comment by miklb** [comment]
That actually clarifies things for me as a fairly new user. I'm wanting to eventually tweet from my site (#indieweb) and wasn't sure how to organize them against regular posts and keep them chronological.

+1
